DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/ToSpeckle/WallConversionToSpeckle.cs

```diff
@@ -47,11 +42,11 @@ ParameterObjectAssigner parameterObjectAssigner
     _parameterObjectAssigner = parameterObjectAssigner;
   }
 
-  public override RevitWall RawConvert(DB.Wall target)
+  public override SOBR.RevitWall RawConvert(DB.Wall target)
   {
-    RevitWall speckleWall = new() { family = target.WallType.FamilyName.ToString(), type = target.WallType.Name };
+    SOBR.RevitWall speckleWall = new() { family = target.WallType.FamilyName.ToString(), type = target.WallType.Name };
```

Comment:
And I guess none of this is going to be null?

Reply:
That's a very good question, I think it's not possible to have a wall without a family name and a wall type